R/gg-plots.R

```diff
@@ -261,10 +261,11 @@ ggally_density <- function(data, mapping, ...) {
 #' @param group_args arguments being supplied to the split-by-color group's \code{\link[ggplot2]{geom_text}()}
 #' @param justify_labels \code{justify} argument supplied when \code{\link[base]{format}}ting the labels
 #' @param align_percent relative align position of the text. When \code{justify_labels = 0.5}, this should not be needed to be set.
-#' @param alignPercent,displayGrid deprecated. Please use their snake-case counterparts.
+#' @param alignPercent,displayGrid `r lifecycle::badge("deprecated")`. Please use their snake-case counterparts.
```

Comment:
Please move this line to end of params (just before `@seealso`)

Reply:
If the docs are already have the param shifted to bottom, then ignore the request

Reply:
I checked. They get shifted to the bottom automagically :)

![image](https://github.com/ggobi/ggally/assets/4500163/57469404-7b07-49bc-8f4e-cdb356b15110)